crates/db_views/src/post_view.rs

```diff
@@ -356,14 +356,19 @@ impl<'a> PostQueryBuilder<'a> {
     query = match self.sort {
       SortType::Active => query
         .then_order_by(
-          hot_rank(post_aggregates::score, post_aggregates::newest_comment_time).desc(),
+          hot_rank(
+            post_aggregates::score,
+            post_aggregates::newest_comment_time_necro,
+          )
+          .desc(),
         )
-        .then_order_by(post_aggregates::newest_comment_time.desc()),
+        .then_order_by(post_aggregates::newest_comment_time_necro.desc()),
```

Comment:
The hot_rank needs to use a "necro-limited" (to 2 days currently) `newest_comment_time`, whereas the new sort needs to ignore that necro-bump. 

To do that I added another column to `PostAggregates`

Reply:
Is there a clearer term we can use instead of "necro"? If not, you should definitely put this explanation into a comment in the code.

Reply:
K added a comment.